### PR TITLE
refactor some tests, remove onboarding tutorial & preferences

### PR DESCRIPTION
**Pull request by fedekunze**
**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added entries in `PENDING.md` with issue # and GitHub username
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: app/src/renderer/components/common/PagePreferences.vue [hunk 1]
```diff
@@ -7,30 +7,26 @@
...
-          id="toggle-onboarding"
-          value="Launch Tutorial"
-          @click.native="setOnboarding()"
-        />
-      </tm-list-item>
       <tm-list-item
         type="field"
         title="Automatically send usage statistics and crash reports"
-        subtitle="to the Voyager development team"
+        subtitle="to the Cosmos Wallet development team"
```
Comment (faboweb) [inline app/src/renderer/components/common/PagePreferences.vue]: nice

File: app/src/renderer/components/common/PagePreferences.vue [hunk 2]
```diff
@@ -40,34 +36,47 @@
...
+          optin: !this.session.errorCollection
+        })
+      }
     }
   }
 }
 </script>
+<style scoped>
+/* TODO: fix style */
+.tm-form-msg {
```
Comment (faboweb) [inline app/src/renderer/components/common/PagePreferences.vue]: @jbibla could you help?

File: app/src/renderer/vuex/modules/session.js [hunk 3]
```diff
@@ -10,11 +10,11 @@ export default () => {
   const ERROR_COLLECTION_KEY = `voyager_error_collection`
 
   const state = {
-    experimentalMode: config.development,
-    insecureMode: config.development,
+    experimentalMode: config.development, // development mode
```
Comment (faboweb) [inline app/src/renderer/vuex/modules/session.js]: nice

File: test/unit/specs/components/common/PagePreferences.spec.js [hunk 4]
```diff
@@ -33,28 +33,72 @@ describe(`PagePreferences`, () => {
...
+              },
+              mockedConnector: false,
+              nodeUrl: `http://localhost:9070`
+            }
+          }
+        }
+      })
+
+      jest.useFakeTimers()
+      wrapper.vm.setErrorCollection() //old style to test timers
```
Comment (faboweb) [inline test/unit/specs/components/common/PagePreferences.spec.js]: let's check it out together

File: app/src/renderer/components/common/PagePreferences.vue (hunk 2 above)
Comment (jbibla) [reply app/src/renderer/components/common/PagePreferences.vue]: yes, this page is ugly. lol. let's merge and i'll fix.

Comment (jbibla) [reply app/src/renderer/components/common/PagePreferences.vue]: sure thing!

Comment (jbibla) [reply app/src/renderer/components/common/PagePreferences.vue]: honestly, i think we should just delete the preferences page at this point and remove it from the toolbar.

it only has node IP which we show in the network-connectivity toolbar and this switch which doesn't even work properly. 

this switch / toggle will appear in the cookie banner. but for now we don't need this page at all.

Comment (faboweb) [reply app/src/renderer/components/common/PagePreferences.vue]: then kill it 🔥

Comment (fedekunze) [reply app/src/renderer/components/common/PagePreferences.vue]: killed

**Pull request merged**